src/Microsoft.Diagnostics.NETCore.Client/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+#if DIAGNOSTICS_RUNTIME
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("common")]
```

Comment:
There are a bunch of InternalsVisibleTo entries defined in the .csproj file. If there isn't a reason to keep this one separate how we add it there?

Reply:
how about adding a comment in the code saying this and then call it good? If later someone wants to try refactoring it a comment in the code will give them the right context.

Reply:
The csproj use of that requires full arcade support at https://github.com/dotnet/arcade/blob/78eaf78761027d225030be2b28aaf4e8bf392929/src/Microsoft.DotNet.Arcade.Sdk/tools/GenerateInternalsVisibleTo.targets#L4 - it's likely something is missing to get it working. Not sure it's worth trying to hook it up as dealing with this in runtime can get non-trivial.

Reply:
Added a comment to provide context